packages/renderer/src/lib/ui/StatusDot.svelte

```diff
@@ -0,0 +1,35 @@
+<!-- StatusDot.svelte -->
+<script lang="ts">
+import type { PodInfoContainerUI } from '../pod/PodInfoUI';
+import Tooltip from './Tooltip.svelte';
+
+export let container: PodInfoContainerUI;
+
+// All the possible statuses that will appear for both Pods and Kubernetes
+const colors: Record<string, string> = {
+  // Podman & Kubernetes
+  running: 'bg-green-500', // Green for running
+
+  // Kubernetes-only
+  terminated: 'bg-red-500', // Red for terminated
+  waiting: 'bg-amber-500', // Amber for waiting
+
+  // Podman-only
+  stopped: 'bg-gray-500', // Gray indicated dormancy / inactivity
+  paused: 'bg-amber-500', // Equivilant to "waiting" in Kubernetes
```

Comment:
-600. and Equivalent :)

Reply:
what about adding a `status` color in the palette (tailwind.config.js) with something like (colors codes are pure example) 

```js
'status': {
        'running': tailwindColors.green[500],
        'stopped': tailwindColors.amber[500],
        etc..
      }
```

so when we need to display a running state we do `bg-status-running` and we see values only at one location

Reply:
or an utility class that provides/share the value across StatusDot and StatusIcon

Reply:
+1 to tailwind cfg, I think we could use these for things like machine status too.